dm/dm/config/subtask.go

```diff
@@ -229,6 +229,10 @@ type SubTaskConfig struct {
 	// if case insensitive, we would convert schema/table name/pattern to lower case
 	CaseSensitive bool `toml:"case-sensitive" json:"case-sensitive"`
 
+	// default "loose" handle create sql by original sql, will not add default collation as upstream
+	// "strict" will add default collation as upstream, and downstream will occur error when downstream don't support
+	CollationCompatible string `yaml:"collation_compatible" toml:"collation_compatible" json:"collation_compatible"`
```

Comment:
could you please add a subtask in https://github.com/pingcap/tiflow/issues/3397  to remind us not to forget to add this new config to OpenAPI

Reply:
ok